supervisor/host/network.py

```diff
@@ -100,6 +103,25 @@ def get(self, inet_name: str) -> Interface:
             self.sys_dbus.network.interfaces[inet_name]
         )
 
+    @Job(conditions=JobCondition.HOST_NETWORK)
```

Comment:
We don't have this in update, is it really required?

Reply:
because why we should do it without a network manager?